vpr/src/place/move_transactions.cpp

```diff
@@ -40,11 +51,11 @@ void apply_move_blocks(const t_pl_blocks_to_be_moved& blocks_affected) {
 
     //Swap the blocks, but don't swap the nets or update place_ctx.grid_blocks
     //yet since we don't know whether the swap will be accepted
-    for (int iblk = 0; iblk < blocks_affected.num_moved_blocks; ++iblk) {
-        ClusterBlockId blk = blocks_affected.moved_blocks[iblk].block_num;
+    for (const auto& block : blocks_affected.moved_blocks) {
```

Comment:
I think if the element type (t_pl_moved_block) is explicitly specified instead of using auto keyword, the code would be more readable, especially that the type name is not that long.

Reply:
I also suggest that the element name changes from "block" to "moved_block"